src/games/supported/YarsRevenge.cpp

```diff
@@ -134,3 +138,41 @@ ActionVect YarsRevengeSettings::getStartingActions() {
     startingActions.push_back(PLAYER_A_FIRE);
     return startingActions;
 }
+
+// returns a list of mode that the game can be played in
+ModeVect YarsRevengeSettings::getAvailableModes(){
+    ModeVect modes;
+    modes.push_back(0);
+    modes.push_back(0x20);
+    modes.push_back(0x40);
+    modes.push_back(0x60);
+    return modes;
+}
+
+// set the mode of the game
+// the given mode must be one returned by the previous function
+void YarsRevengeSettings::setMode(game_mode_t m, System &system, StellaEnvironment& environment){
+    if(m == 0 || m == 0x20 || m == 0x40 || m == 0x60 ){
```

Comment:
The manual states some differences between mode 0 and 0x20, but I can't see these visually. Do you remember what the differences are?